config/featureflags.ts

```diff
@@ -2,10 +2,19 @@
  * Copyright Strimzi authors.
  * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
  */
-import { Config } from './types';
+import { Config } from './config.types';
 
 /**
  * feature flag configuration - used to enable/disable capabilities across the UI/server. The values can be static or dynamic, but the value must always be boolean
  */
 
-export const featureFlags: Config<boolean> = {};
+export const featureFlags: Config<boolean> = {
+  client: {
+    Home: {
```

Comment:
Same concern over having to define these (they're more like `ConfigValue<boolean>`)

Reply:
It feels like the UI should not have to know about the flag structure to query for it - and same with server, it parses some feature flag config and then has a key:value structure rather than having to define each feature flag.

Reply:
I think it's better have an explicit schema for the feature flags (which I think this is providing). A freeform map is quite hard to manage in practice, as you don't know the total set of flags you have in the app.